docs/reference/formulas/smartwatts.md

```diff
@@ -1,17 +1,13 @@
 # SmartWatts Formula
 
-SmartWatts is a software-defined power meter based on the PowerAPI toolkit.
-SmartWatts is a configurable software that can estimate the power consumption of
-software in real-time.
-SmartWatts needs to receive several metrics provided by
-[HWPC Sensor](../sensors/hwpc-sensor.md#events) :
+SmartWatts is a Formula, a configurable software that can estimate the power consumption of software in real-time.
+SmartWatts needs to receive several metrics available in [Reports](../report/report.md), [HWPC Sensor](../sensors/hwpc-sensor.md#events) is a Sensor compatible (i.e making the necessary metrics available in HWPC Reports):
```

Comment:
SmartWatts required `HWPC Reports`. I agree that the sensor is just a possible source of this kind of reports. However, regarding the sentence, one can say the `Power Reports` are also accepted by the formula